Sim/SimG4Components/src/SimG4ConstantMagneticFieldTool.cpp

```diff
@@ -48,9 +48,9 @@ StatusCode SimG4ConstantMagneticFieldTool::initialize() {
         new sim::ConstantField(m_fieldComponentX, m_fieldComponentY, m_fieldComponentZ, m_fieldRadMax, m_fieldZMax);
     fieldManager->SetDetectorField(m_field);
 
-    fieldManager->CreateChordFinder(m_field);
-    G4ChordFinder* chordFinder = fieldManager->GetChordFinder();
-    chordFinder->GetIntegrationDriver()->RenewStepperAndAdjust(stepper(m_integratorStepper, m_field)); 
+    G4ChordFinder* chordFinder = 
+    new G4ChordFinder( m_field,  1.0e-2 * mm, stepper(m_integratorStepper, m_field));
```

Comment:
What's this 1.0e-2?

Reply:
That's the default value for `stepMinimum`, see:

 I had to set this in order to use the ctor with the third argument, as `RenewStepperAndAdjust` doesn't work anymore with the geant4 version in the new lcg release.

Reply:
I can add a comment, or also a parameter for the algorithm.

Reply:
yes, please add a comment and if needed we we can had a parameter later on.

Reply:
Done. Also added the ExactHelixStepper, which is the recommended one for a constant magnetic field.